### PR TITLE
Fix non unique typename

### PR DESCRIPTION
**Pull request by C-R-W**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Fixed tickets | #...   

Fix non unique type name.

Bug:
If there is more then one SerializedDataType in a schema entity a exception gets thrown.

Uncaught PHP Exception GraphQL\Error\InvariantViolation: "Schema must contain unique named types but contains multiple types named "coreShopSerializedData" (see http://webonyx.github.io/graphql-php/type-system/#type-registry)." at /var/www/vhosts/portal.prooptik.de/pimcore-live/vendor/webonyx/graphql-php/src/Utils/Utils.php line 333 {"exception":"[object] (GraphQL\\Error\\InvariantViolation(code: 0): Schema must contain unique named types but contains multiple types named \"coreShopSerializedData\" (see http://webonyx.github.io/graphql-php/type-system/#type-registry). at /var/www/vhosts/portal.prooptik.de/pimcore-live/vendor/webonyx/graphql-php/src/Utils/Utils.php:333)"

### CODE REVIEW

**Comment by dpfaffenbauer** [comment]
@C-R-W please sign the CLA and check why tests are failing now

File: src/CoreShop/Bundle/PimcoreBundle/DataHub/Type/SerializedDataType.php [hunk 1]
```diff
@@ -15,9 +15,12 @@
 namespace CoreShop\Bundle\PimcoreBundle\DataHub\Type;
 
 use GraphQL\Type\Definition\ScalarType;
+use Pimcore\Bundle\DataHubBundle\GraphQL\DataObjectType\GeopointType;
```
Comment (dpfaffenbauer) [inline src/CoreShop/Bundle/PimcoreBundle/DataHub/Type/SerializedDataType.php:18]: not needed here. pelase remove

File: src/CoreShop/Bundle/PimcoreBundle/DataHub/QueryType/SerializedData.php [hunk 2]
```diff
@@ -36,6 +36,6 @@ public function getGraphQlFieldConfig($attribute, Data $fieldDefinition, $class
 
     public function getFieldType(Data $fieldDefinition, $class = null, $container = null)
     {
-        return new SerializedDataType();
+        return SerializedDataType::getInstance();
```
Comment (dpfaffenbauer) [inline src/CoreShop/Bundle/PimcoreBundle/DataHub/QueryType/SerializedData.php:39]: don't use a singleton here, rather create the instance here and keep it.

**Review by dpfaffenbauer — CHANGES_REQUESTED** [review]

**Comment by dpfaffenbauer** [comment]
@C-R-W Tests fail for other reasons, I still have some comments

**Comment by C-R-W** [comment]
Hi, i updated the files, is this ok so?

File: src/CoreShop/Bundle/PimcoreBundle/DataHub/QueryType/SerializedData.php [hunk 3]
```diff
@@ -15,11 +15,14 @@
 namespace CoreShop\Bundle\PimcoreBundle\DataHub\QueryType;
 
 use CoreShop\Bundle\PimcoreBundle\DataHub\Type\SerializedDataType;
+use phpDocumentor\Reflection\Types\Self_;
```
Comment (dpfaffenbauer) [inline src/CoreShop/Bundle/PimcoreBundle/DataHub/QueryType/SerializedData.php:18]: what is this?

**Review by dpfaffenbauer — CHANGES_REQUESTED** [review]

File: src/CoreShop/Bundle/PimcoreBundle/DataHub/QueryType/SerializedData.php (hunk 3 above)
Comment (C-R-W) [reply src/CoreShop/Bundle/PimcoreBundle/DataHub/QueryType/SerializedData.php]: sorry, just removed it.

**Review by dpfaffenbauer — APPROVED** [review]

**Comment by dpfaffenbauer** [comment]
thanks @C-R-W

**Pull request merged**